lms/djangoapps/teams/search_indexes.py

```diff
@@ -0,0 +1,92 @@
+""" Search index used by haystack to load data into elasticsearch"""
+
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from search.search_engine_base import SearchEngine
+
+from .serializers import CourseTeamSerializer, CourseTeam
+
+
+class CourseTeamIndexer(object):
+    """
+    Index object for CourseTeam model instances
+    """
+    INDEX_NAME = "course_team_index"
+    DOCUMENT_TYPE_NAME = "course_team"
+    ENABLE_INDEXING_KEY = "ENABLE_TEAMS_INDEX"
+
+    def __init__(self, course_team):
+        self.course_team = course_team
+
+    def data(self):
+        """
+        Uses the CourseTeamSerializer to create a serialized course_team object
+        Adds in additional fields specific to search
+
+        Returns serialized object with additional search fields
+        """
+        serialized_course_team = CourseTeamSerializer(self.course_team).data
+        serialized_course_team['pk'] = self.course_team.pk
```

Comment:
I can change it back to `id`.  Using haystack it needed to be `pk` because `id` was a custom elasticsearch id.  But now that we are doing it this way we can use `id` again